src/backend/runner-base.js

```diff
@@ -94,16 +94,23 @@ module.exports = class BaseRunner {
 	/**
 	 * @memberOf BaseRunner#
 	 * @function
-	 * 
+	 *
 	 * @name buildKernel
-	 * 
+	 *
 	 * @desc Get and returns the Synchronous executor, of a class and kernel
 	 * Which returns the result directly after passing the arguments.
-	 * 
+	 *
 	 */
 	buildKernel(fn, settings) {
 		settings = Object.assign({}, settings || {});
-		const fnString = fn.toString();
+
+		var fnString = '';
```

Comment:
`const fnString = fn.toString();` should be sufficient on a string or a function.

Reply:
I will do that